charts/fpl-case-service/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 name: fpl-case-service
 apiVersion: v1
 home: https://github.com/hmcts/fpl-ccd-configuration
-version: 1.11.5
+version: 1.11.6
```

Comment:
do we need this and the other chart changes here?

Reply:
this configured preview to use the same ccd images as prod which let us test that it fixed the problem; I think we can merge the chart changes with this PR and close the other one